src/views/virtualmachines/actions/components/ConfirmMultipleVMActionsModal/ConfirmMultipleVMActionsModal.tsx

```diff
@@ -56,7 +56,7 @@ const ConfirmMultipleVMActionsModal: FC<ConfirmMultipleVMActionsModalProps> = ({
         title={t('{{actionType}} {{numVMs}} VirtualMachines?', { actionType, numVMs })}
       />
       <ModalBody>
-        {t('Are you sure you want to stop ')}
+        {t('Are you sure you want to {{actionType}} ', { actionType: actionType.toLowerCase() })}
```

Comment:
i dont see action type also translated, so we will end up with a sentence in split language

Reply:
splitting the phrases?